v2/pink-sb/src/lib/action-list/Anchor.svelte

```diff
@@ -30,11 +32,19 @@
         {/if}
     </Stack>
 </a>
+{#if hasDivider}
+    <div class="divider-container">
```

Comment:
we should probably just start allowing `styles` and `classes` from top to bottom on components itself to avoid these lol.